portality/models/article.py

```diff
@@ -271,6 +292,13 @@ def _generate_index(self):
         issns += cbib.get_identifiers(cbib.P_ISSN)
         issns += cbib.get_identifiers(cbib.E_ISSN)
 
+        # get the issn from the journal bibjson
+        if type(cbib.journal_issns) is list:
```

Comment:
```isinstance(cbib.journal_issns, list)``` is the canonical way to check (also includes child classes, whereas this line is doing an exact match, i.e. won't work for list-like-collections). See http://stackoverflow.com/a/152596/1154882 if you'd like a bit more background.